README.md

```diff
@@ -11,9 +11,9 @@ Offers a command-line helper tool, `tf`
 
 YleTf requires Terraform which can be installed according to their [instructions](https://www.terraform.io/intro/getting-started/install.html). On MacOS (and OSX) you can use [Homebrew](https://brew.sh/). You can also easily install and manage multiple versions of Terraform with [homebrew-terraforms](https://github.com/Yleisradio/homebrew-terraforms).
 
-YleTf supports Terraform versions from 0.9 on. Tested versions range from 0.11 to 1.0.
+YleTf supports Terraform versions from 0.11 on. Tested versions range from 0.11 to 1.0.
```

Comment:
I think everything should still work with v0.9. But I think it's fine not to promise anything, and hopefully no-one is still using such old versions anyway.